pkg/controller/clusterprovision/installlogmonitor_test.go

```diff
@@ -262,6 +263,26 @@ func TestParseInstallLog(t *testing.T) {
 			}},
 			expectedReason: "DNSAlreadyExists",
 		},
+		{
+			name: "GCP compute quota",
+			log:  pointer.StringPtr(gcpCPUQuotaLog),
+			existing: []runtime.Object{&corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      regexConfigMapName,
+					Namespace: constants.DefaultHiveNamespace,
+				},
+				Data: map[string]string{
+					"regexes": `
+- name: GCPComputeQuota
```

Comment:
I can confirm that you need to copy the yaml stanza in here to make the test work... which sucks. Good future improvement to the test suite to load up the actual yaml file instead, so there's no possibility of drift.